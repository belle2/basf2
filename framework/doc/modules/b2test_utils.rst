.. automodule:: b2test_utils
   :members:
