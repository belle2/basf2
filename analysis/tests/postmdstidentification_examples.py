#!/usr/bin/env python3
# -- analysis/tests/postmdstidentification_examples.py

##########################################################################
# basf2 (Belle II Analysis Software Framework)                           #
# Author: The Belle II Collaboration                                     #
#                                                                        #
# See git log for contributors and copyright holders.                    #
# This file is licensed under LGPL-3.0, see LICENSE.md.                  #
##########################################################################

import b2test_utils as b2tu
import b2test_utils_analysis as b2tua

#: extend the examples test class to actually run over the post mdst identification examples:


class ExamplesTest(b2tua.ExamplesTest):
    def test_postmdstidentification_examples(self):
        """
        Test supported PostMdstIdentification examples.
        """

        self._test_examples_dir('analysis/examples/PostMdstIdentification/')


if __name__ == '__main__':
    with b2tu.clean_working_directory():
        b2tua.unittest.main()
