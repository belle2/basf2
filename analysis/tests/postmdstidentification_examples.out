[INFO] Steering file: analysis/tests/postmdstidentification_examples.py
TFile: ChargedPidMVAModule.root
TTree: my_deuteron
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 10
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: pidChargedBDTScore__bo11__cm__spALL__bc, 0.02931
TBranch: pidChargedBDTScore__bo13__cm__spALL__bc, 0.002839
TBranch: pidChargedBDTScore__bo211__cm__spALL__bc, 0.1905
TBranch: pidChargedBDTScore__bo321__cm__spALL__bc, 0.7773
TBranch: pidChargedBDTScore__bo2212__cm__spALL__bc, nan
TBranch: pidChargedBDTScore__bo1000010020__cm__spALL__bc, nan
TTree: my_p
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 10
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: pidChargedBDTScore__bo11__cm__spALL__bc, 0.02931
TBranch: pidChargedBDTScore__bo13__cm__spALL__bc, 0.002839
TBranch: pidChargedBDTScore__bo211__cm__spALL__bc, 0.1905
TBranch: pidChargedBDTScore__bo321__cm__spALL__bc, 0.7773
TBranch: pidChargedBDTScore__bo2212__cm__spALL__bc, nan
TBranch: pidChargedBDTScore__bo1000010020__cm__spALL__bc, nan
TTree: my_K
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 10
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: pidChargedBDTScore__bo11__cm__spALL__bc, 0.02931
TBranch: pidChargedBDTScore__bo13__cm__spALL__bc, 0.002839
TBranch: pidChargedBDTScore__bo211__cm__spALL__bc, 0.1905
TBranch: pidChargedBDTScore__bo321__cm__spALL__bc, 0.7773
TBranch: pidChargedBDTScore__bo2212__cm__spALL__bc, nan
TBranch: pidChargedBDTScore__bo1000010020__cm__spALL__bc, nan
TTree: my_pi
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 10
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: pidChargedBDTScore__bo11__cm__spALL__bc, 0.02931
TBranch: pidChargedBDTScore__bo13__cm__spALL__bc, 0.002839
TBranch: pidChargedBDTScore__bo211__cm__spALL__bc, 0.1905
TBranch: pidChargedBDTScore__bo321__cm__spALL__bc, 0.7773
TBranch: pidChargedBDTScore__bo2212__cm__spALL__bc, nan
TBranch: pidChargedBDTScore__bo1000010020__cm__spALL__bc, nan
TTree: my_mu
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 10
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: pidChargedBDTScore__bo11__cm__spALL__bc, 0.02405
TBranch: pidChargedBDTScore__bo13__cm__spALL__bc, 0.1466
TBranch: pidChargedBDTScore__bo211__cm__spALL__bc, 0.828
TBranch: pidChargedBDTScore__bo321__cm__spALL__bc, 0.001307
TBranch: pidChargedBDTScore__bo2212__cm__spALL__bc, nan
TBranch: pidChargedBDTScore__bo1000010020__cm__spALL__bc, nan
TTree: my_e
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 10
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: pidChargedBDTScore__bo11__cm__spALL__bc, 0.02405
TBranch: pidChargedBDTScore__bo13__cm__spALL__bc, 0.1466
TBranch: pidChargedBDTScore__bo211__cm__spALL__bc, 0.828
TBranch: pidChargedBDTScore__bo321__cm__spALL__bc, 0.001307
TBranch: pidChargedBDTScore__bo2212__cm__spALL__bc, nan
TBranch: pidChargedBDTScore__bo1000010020__cm__spALL__bc, nan
TFile: LowEnergyPi0Identification.root
TTree: t1
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 38
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: px, 0.04916
TBranch: py, 0.2831
TBranch: pz, -0.1112
TBranch: pt, 0.2873
TBranch: p, 0.3081
TBranch: E, 0.3335
TBranch: InvM, 0.1277
TBranch: identification, 0.187
